src/Move.jsx

```diff
@@ -102,19 +102,20 @@ export default class Move extends Component {
 
 					const { node, transform } = data;
 
-					node.style.transition = 'opacity 160ms, transform 260ms ease-out';
-
 					if (this.createQueue.indexOf(key) >= 0) {
-						node.style.opacity = 1;
+                        node.style.transition = 'opacity 160ms, transform 260ms ease-out';
```

Comment:
Some indent issue.

Maybe keep repeating style definition in variable?
```
const transitionStyle = 'opacity 160ms, transform 260ms ease-out';
node.style.transition = transitionStyle;
```

Reply:
yeah ;)